src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerConstants.java

```diff
@@ -124,4 +124,7 @@ private CaseWorkerConstants() {
     public static final String AND = "and";
 
     public static final String RESPONSE_BODY_MISSING_FROM_UP = "Response Body is missing from User Profile";
+
+    public static final String NAME_REGEX = "^[a-zA-Z' -]*$";
```

Comment:
Regex will only allow `a-z`, `A-Z`, `'`, ` `, `-` characters 

_This regex will be aligned to HMCTS standard in a later ticket_